src/main/js/component/common/RefreshTableCellFormatter.js

```diff
@@ -17,6 +18,7 @@ const RefreshTableCellFormatter = ({ handleButtonClicked, currentRowSelected })
 );
 
 RefreshTableCellFormatter.propTypes = {
+    id: PropTypes.string.isRequired,
```

Comment:
Why choose to require this one?

Reply:
Because the original has no default values whereas IconTableCellFormatter does have default values.  I will add default values for this class.